DependencyInjection/DoctrineExtension.php

```diff
@@ -643,7 +536,7 @@ protected function ormLoad(array $config, ContainerBuilder $container)
         $container->registerForAutoconfiguration(AbstractIdGenerator::class)
             ->addTag(IdGeneratorPass::ID_GENERATOR_TAG);
 
-        if (method_exists($container, 'registerAttributeForAutoconfiguration')) {
+        if (PHP_VERSION_ID >= 80000 && method_exists(ContainerBuilder::class, 'registerAttributeForAutoconfiguration')) {
```

Comment:
Why is this change necessary?

Reply:
Psalm was complaining with `RedundantCondition` :man_shrugging: I aligned this with https://github.com/doctrine/DoctrineBundle/blob/4bbe5880469d3cfab660f4dd887959b22bcf9dc4/DependencyInjection/DoctrineExtension.php#L162

Reply:
That's a false positive. Psalm analyzes the code with Symfony 6 deps installed and this condition makes sure we don't run into errors with Symfony 4. We should rather suppress the error here.

Reply:
Ok I don't mind. Changed it back.